### PR TITLE
AT_01.01_033 | Validate <Header> head icon

### PR DESCRIPTION
**Pull request by Olgret**
https://trello.com/c/0uCxm7UO

### CODE REVIEW

**Review by olenamio — COMMENTED** [review]
dear Olga, code line with cy.visit('http://localhost:8080/') looks to be unnecessary to add. with beforeEach method we are already at this stage.

File: cypress/e2e/headerHeadIcon.cy.js [hunk 1]
```diff
@@ -12,4 +12,17 @@ describe('Header Head Icon', () => {
...
+
+    it('AT_01.01_033 | Validate <Header> head icon', () => {
+        cy.visit('http://localhost:8080/');
+        cy.get('div h1').should('have.text','Welcome to Jenkins!')
+                        .should('be.visible');
+        cy.get('span.task-link-text').contains('People').click({force: true});
+        cy.get('div h1').should('have.text','\n            People\n            ')
+                        .should('exist');
+        cy.get('#jenkins-home-link').click();
+        cy.get('div h1').should('have.text','Welcome to Jenkins!')
```
Comment (Rom-T) [inline cypress/e2e/headerHeadIcon.cy.js:24]: When we make several assertions for 1 element, only the first assertion `should`, and all subsequent ones are written as `and`

**Comment by Olgret** [comment]
> dear Olga, code line with cy.visit('http://localhost:8080/') looks to be unnecessary to add. with beforeEach method we are already at this stage.

Thanks for the review, I removed the unnecessary code.

File: cypress/e2e/headerHeadIcon.cy.js (hunk 1 above)
Comment (Olgret) [reply cypress/e2e/headerHeadIcon.cy.js:21]: Thank you very much for the review, I removed the unnecessary code and changed assertions!

**Review by matvlesia — APPROVED** [review]
I think you can replace {force: true} with realHover() as Maria showed in her lesson

**Review by Rom-T — APPROVED** [review]

**Pull request merged**